Cert-manager webhook doesn't have the CA injected yet

https://github.com/scylladb/scylla-operator/runs/2768929258?check_suite_focus=true#step:9:173

```
Error from server (InternalError): error when creating "/home/runner/work/_temp/e2e-artifacts/deploy/operator/10_certificate.yaml": Internal error occurred: failed calling webhook "webhook.cert-manager.io": Post "https://cert-manager-webhook.cert-manager.svc:443/mutate?timeout=10s": x509: certificate signed by unknown authority
Error from server (InternalError): error when creating "/home/runner/work/_temp/e2e-artifacts/deploy/operator/10_issuer.yaml": Internal error occurred: failed calling webhook "webhook.cert-manager.io": Post "https://cert-manager-webhook.cert-manager.svc:443/mutate?timeout=10s": x509: certificate signed by unknown authority
Error: Process completed with exit code 1.
```

--- a/hack/ci-deploy.sh
+++ b/hack/ci-deploy.sh
@@ -40,6 +40,7 @@ for d in cert-manager{,-cainjector,-webhook}; do
     wait-for-object-creation cert-manager deployment.apps/"${d}"
     kubectl -n cert-manager rollout status --timeout=5m deployment.apps/"${d}"
 done
+wait-for-object-creation cert-manager secret/cert-manager-webhook-ca
 
 kubectl apply -f "${deploy_dir}"/operator
 

